libnopegl/src/scene.c

```diff
@@ -267,6 +267,27 @@ int ngl_scene_get_filepaths(struct ngl_scene *s, char ***filepathsp, size_t *nb_
     return 0;
 }
 
+static void update_file_back_ref(struct ngl_scene *s, size_t index, const char *str)
+{
+    const char **filep = ngli_darray_get(&s->files, index);
+    *filep = str;
+}
+
+void ngli_scene_update_file_back_ref(struct ngl_node *node, const struct node_param *par)
```

Comment:
What about `ngli_scene_update_filepath_ref` to make it consistent with `ngl_scene_update_filepath` and `_ref` is enough.

Reply:
Renamed